servicetalk-encoding-netty/src/main/java/io/servicetalk/encoding/netty/NettyCompressionStreamingSerializer.java

```diff
@@ -110,6 +126,34 @@ public void onComplete() {
 
                 subscriber.onComplete();
             }
+
+            private void aggregateNext(Buffer next) {
+                assert aggregate != null;
+                if (!aggregate.isReadOnly() && aggregate.maxWritableBytes() >= next.readableBytes()) {
+                    aggregate.writeBytes(next);
+                } else {
+                    Buffer newAgg = allocator.newBuffer(aggregate.readableBytes() + next.readableBytes(),
+                            Integer.MAX_VALUE);
```

Comment:
This is equal to the default `maxCapacity`, consider using an overload without 2nd argument.

Reply:
unless its guaranteed by the interface I will keep it (I don't think we need to guarantee this in the interface) bcz this use case is specifically using the buffer for aggregation